presto-main/src/main/java/com/facebook/presto/execution/SqlQueryManager.java

```diff
@@ -48,6 +48,7 @@
 import com.facebook.presto.sql.tree.Statement;
 import com.facebook.presto.transaction.TransactionManager;
 import com.facebook.presto.util.StatementUtils;
+import com.facebook.presto.version.EmbedVersion;
```

Comment:
Could you attach an example stacktrace?

Reply:
Sure. I pasted it in https://github.com/prestodb/presto/pull/11700#issue-222447112